WebClock/CountBalance.cs

```diff
@@ -27,5 +27,42 @@ public static BalanceDto CountWorkTime(List<ClockInOut> datesIn, List<ClockInOut
                 MinutesLeft = totalLeftTime.Minutes
             };
         }
+        public static BalanceDto CountWorkTimeCurrent(List<ClockInOut> datesIn, List<ClockInOut> datesOut)
+        {
+            var totalWorkTime = new TimeSpan();
+            var hoursToWorkInMonthToThisDay = new TimeSpan(0, Convert.ToInt32(GetBusinessDaysInThisMonth() * 8), 0, 0);
+            if (datesIn.Count > datesOut.Count)
+            {
+                datesOut.Add(new ClockInOut { UserId = 1, Date = DateTime.UtcNow, Type = ClockType.Out });
+            }
+            for (int i = 0; i < datesIn.Count; i++)
+            {
+                totalWorkTime += datesOut[i].Date.TimeOfDay - datesIn[i].Date.TimeOfDay;
+            }
+
+            var totalLeftTime = hoursToWorkInMonthToThisDay - totalWorkTime;
+
+            return new BalanceDto
+            {
+                HoursWorked = (int)totalWorkTime.TotalHours,
+                MinutesWorked = totalWorkTime.Minutes,
+                HoursLeft = (int)totalLeftTime.TotalHours,
+                MinutesLeft = totalLeftTime.Minutes
+            };
+
+        }
+        public static double GetBusinessDaysInThisMonth()
+        {
+            DateTime startD = new DateTime(DateTime.UtcNow.Year, DateTime.UtcNow.Month, 1, 0, 0, 0);
```

Comment:
proponuję pełną nazwę: startDate - to tylko trzy litery więcej, ale już nie ma wątpliwości o co chodzi.